src/client/src/rt-storybook/Story/Story.tsx

```diff
@@ -1,9 +1,54 @@
 import React, { Component } from 'react'
-import { ThemeSwitcher } from 'rt-components'
-import { styled } from 'rt-util'
-import { Themes } from 'shell/theme'
-import 'ui/styles/css/index.css'
-import Theme from 'ui/theme/Theme'
+import styled from 'react-emotion'
+import { ThemeState } from 'rt-theme'
+
+class Story extends Component {
+  render() {
+    const { children } = this.props
+
+    return (
+      <ThemeState.Provider name="light">
+        <StyledStory>
+          <Toolbar>
+            <ThemeState.Consumer>
+              {({ name, setTheme }) => (
+                <IconButton
+                  onClick={() => setTheme({ name: name === 'dark' ? 'light' : 'dark' })}
+                  type={name || 'primary'}
```

Comment:
Where is this being used?

Reply:
It's copied from `<ThemeSwitcher/>`, and `ThemeSwicher` was removed.